app/controllers/registers_controller.rb

```diff
@@ -2,8 +2,18 @@
 
 class RegistersController < ApplicationController
   def index
-    @search = Register.available.ransack(params[:q])
-    @registers = @search.result.where(register_phase: 'Beta').sort_by_phase_name_asc.by_name
+    if params[:q].present?
+      @registers = Register.available.in_beta.search_for(params[:q])
+    elsif params[:sort].present?
```

Comment:
I think this logic can be moved into the model with an optional parameter. i.e. the pattern in: https://github.com/openregister/registers-frontend/blob/2bff975b668152d0858fc1e1a72a1860c86bcffb/app/models/concerns/search_scope.rb#L4